opcode.h

```diff
@@ -0,0 +1,34 @@
+/* File automatically generated by scripts/gen_opcode.py. */
```

Comment:
Since `opcode.h` is generated by Python script, we don't have to track this file in GIT repository.